lib/src/signed_video_h26x_sign.c

```diff
@@ -615,11 +616,45 @@ SignedVideoReturnCode
 signed_video_get_sei(signed_video_t *self, uint8_t *sei, size_t *sei_size)
 {
 
+  svrc_t status = SV_UNKNOWN_FAILURE;
```

Comment:
This copied code snippet is pasted in two places. You should try to turn it into a function to avoid divergence between them in the future.